module/applications/actor/sheet-v2-mixin.mjs

```diff
@@ -0,0 +1,633 @@
+import * as Trait from "../../documents/actor/trait.mjs";
+import { formatNumber, simplifyBonus, staticID } from "../../utils.mjs";
+import Tabs5e from "../tabs.mjs";
+
+/**
+ * Adds common V2 sheet functionality.
+ * @param {typeof ActorSheet5e} Base  The base class being mixed.
+ * @returns {typeof ActorSheetV2}
+ */
+export default function ActorSheetV2Mixin(Base) {
+  return class ActorSheetV2 extends Base {
+    /**
+     * @typedef {object} SheetTabDescriptor5e
+     * @property {string} tab     The tab key.
+     * @property {string} label   The tab label's localization key.
+     * @property {string} [icon]  A font-awesome icon.
+     * @property {string} [svg]   An SVG icon.
+     */
+
+    /**
+     * Sheet tabs.
+     * @type {SheetTabDescriptor5e[]}
+     */
+    static TABS = [];
+
+    /**
+     * Available sheet modes.
+     * @enum {number}
+     */
+    static MODES = {
+      PLAY: 1,
+      EDIT: 2
+    };
+
+    /**
+     * The mode the sheet is currently in.
+     * @type {ActorSheetV2.MODES}
+     * @protected
+     */
+    _mode = this.constructor.MODES.PLAY;
+
+    /**
+     * The cached concentration information for the character.
+     * @type {{items: Set<Item5e>, effects: Set<ActiveEffect5e>}}
+     * @internal
+     */
+    _concentration;
+
+    /* -------------------------------------------- */
```

Comment:
Might be nice to have some separators to distinguish the grouping of methods (e.g. "Rendering" & "Event Listeners & Handlers").

Would involve shifting the `activateListeners` method beneath the other preparation methods.